### PR TITLE
added support for dj 1.9.x

### PR DESCRIPTION
**Pull request by piotrpawlaczek**
This branch works with django 1.8.x and django 1.9.x
For older django versions stick with something different or feel free to commit a patch.

Changes:
- added required from_db_value method
- slightly refactored code

### CODE REVIEW

**Comment by palewire** [comment]
Thanks for the patch. Looks like a couple tests are failing over at TravisCI, but I haven't investigated why yet. I'll give this a look when I have a moment but you know the key feel free to patch away.

**Comment by piotrpawlaczek** [comment]
In dj 1.9 they'd drop support for python 3.2 and 3.3 so tests obviously fails with py3.3 + dj1.9 combination.

Although the patch adds support for django 1.9 and works with django 1.8, it breaks compatibility with the older versions. OrdererdDict<>Text conversion completely fails for dj <= 1.7 same as your master branch with dj 1.9 

If you know what's the cause I'd be happy to improve....

**Comment by palewire** [comment]
Django's official documentation says they are dropping support for anything prior to 1.7 so I think we can do the same. I just updated the Travis configuration file in the master branch to drop the old ones and add 1.9 beta. I also added Python 3.5 and dropped 3.2 and 3.3.

Thanks for the prompt on this. It had simply slipped behind. What do you think of this new test matrix? If you like it, maybe try pulling it into your patch and see if we can get those to pass.

File: setup.py [hunk 1]
```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```
Comment (funkybob) [inline setup.py]: Surely this should remain at 1.7... otherwise people still using 1.7 [like the person who linked me to this PR] will get failures.

Comment (piotrpawlaczek) [inline setup.py]: @palewire 
On django 1.7.10 some tests fail.

**new_obj.yaml** is not converted to **python obj** but remains **unicode**. Therefore the following tests fail:

- test_blank_yaml_field
- test_nested_ordereddict
- test_ordereddict
- test_yaml_field_create
- test_yaml_field_load
- test_yaml_list

Once we have this fixed, we can change the condition mentioned by @funkybob  to include 1.7 as well. 

On django 1.8.x and django 1.9rc1 everything works like a charm.

Comment (palewire) [inline setup.py]: I'm not sure I totally follow. Are you saying there is a bug in the patch, or in the tests?

Comment (funkybob) [inline setup.py]: This guard is only for calling django.setup() .. which is required in 1.7 as well as 1.8.

Comment (piotrpawlaczek) [inline setup.py]: @palewire 
The patch simply doesn't work with dj 1.7 regardless of the guard. 
Tests are fine.

**Comment by palewire** [comment]
This has been handled by #14

**Pull request closed without merging**